core/src/main/java/com/scalar/db/common/CoreError.java

```diff
@@ -736,6 +736,18 @@ public enum CoreError implements ScalarDbError {
       "The BOOLEAN type is not supported for index columns in DynamoDB. Column: %s",
       "",
       ""),
+  DB2_INDEX_OR_KEY_ON_BLOB_COLUMN_NOT_SUPPORTED(
```

Comment:
For consistency:
```suggestion
  JDBC_DB2_INDEX_OR_KEY_ON_BLOB_COLUMN_NOT_SUPPORTED(
```

Reply:
Thanks
https://github.com/scalar-labs/scalardb/pull/3000/commits/897a4b6600b0703e3f17d43541d3f59287388c8c